packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -500,10 +500,11 @@ export class CalloutContentBase extends React.Component<ICalloutProps, ICalloutS
         const currentDoc: Document = getDocument(currentElement)!;
         this._target = currentDoc ? (currentDoc.querySelector(target) as Element) : null;
         this._targetWindow = getWindow(currentElement)!;
-      } else if ((target as MouseEvent).stopPropagation) {
-        this._targetWindow = getWindow((target as MouseEvent).toElement as HTMLElement)!;
+      } else if (!!(target as MouseEvent).stopPropagation) {
+        // TODO: vet and test this change!
+        this._targetWindow = getWindow((target as MouseEvent).target as HTMLElement)!;
```

Comment:
@joschect `toElement` was previously deprecated and completely removed in types for TS3.7, so I was hoping you could sanity check these changes for me. Thanks!

Reply:
@JasonGore I do not know why I first used that instead of target! That's a good catch from typescript. Thanks for fixing that!